src/main.cpp

```diff
@@ -9,7 +9,7 @@ int main(int argc, char **argv)
     auto test = new std::string;  // :)
     std::cout << "Hello xD\n";
     auto test2 = new std::string("xd");  // :)
-    auto test3 = new std::string;        // :)
+    auto test3 = new std::string("a");   // :)
```

Comment:
warning: 'auto test3' can be declared as 'auto *test3' [llvm-qualified-auto]

```suggestion
    auto *test3 = new std::string("a");   // :)
```